CMake/PrepareThreadingOptions.cmake

```diff
@@ -12,7 +12,7 @@
 # build, depending on the dependencies found.
 #######################################################################
 
-set(OGRE_THREAD_DEFAULT_PROVIDER "none")
+set(OGRE_THREAD_DEFAULT_PROVIDER "std")
```

Comment:
this requires c++11 compiler support, while OGRE 1.10 is still at C++03. Therefore we can not change it ATM. Also see #454. You might not want to bother unless you need Ogre::WorkQueue.